README.md

```diff
@@ -199,6 +200,9 @@ pipeline.fit(X_train, Y_train)
 predictions = pipeline.predict(X_test)
 ```
 
+## Explaining Hierarchical Classifiers
```

Comment:
This new section is missing a hyperlink in the quick links in the beginning of the README

Reply:
Resolved by https://github.com/scikit-learn-contrib/hiclass/pull/106/commits/af7e83abbebd90e14fbc9abf380d4c68eada4e92.